src/components/App/index.tsx

```diff
@@ -9,7 +10,19 @@ const queryClient = new QueryClient();
 function App() {
   return (
     <QueryClientProvider client={queryClient}>
-      <Signup />;
+      <Router>
+        <Switch>
+          <Route path="/login">
+            <h1>Login Page</h1>
+          </Route>
+          <Route path="/signup">
+            <Signup />
+          </Route>
+          <Route path="/">
+            <Redirect to="/signup" />
+          </Route>
+        </Switch>
+      </Router>
```

Comment:
it is a good practice to put the Router in a `Router` general component, and add all the path constants there.

Also there is a bug in your code, if i am logged in i can access to `login` and `signup` that is wrong. and also if im logged in i will be always redirected to `/signup` no matter what route i go that is not `login` or `signup`